src/demo/level.rs

```diff
@@ -10,17 +10,10 @@ pub(super) fn plugin(_app: &mut App) {
     // later if needed.
 }
 
-#[derive(Debug)]
-pub struct SpawnLevel;
-
-impl Command for SpawnLevel {
-    fn apply(self, world: &mut World) {
-        // The only thing we have in our level is a player,
-        // but add things like walls etc. here.
-        world.commands().add(SpawnPlayer { max_speed: 400.0 });
-
-        // Flush the commands we just added so that they are
-        // all executed now, as part of this command.
-        world.flush_commands();
-    }
+/// A [`Command`] to spawn the level.
+/// Exclusive systems, i.e. systems that only accept `&mut World`, implement [`Command`].
```

Comment:
This is not exactly true: https://docs.rs/bevy_ecs/0.14.1/src/bevy_ecs/system/commands/mod.rs.html#1173-1175

Also I don't think this comment is necessary.

Reply:
Considering how much newcomers were struggling with custom commands, I think it would be even more confusing to have two completely different ways of defining them without explanation.

Reply:
I thought the struggle with custom commands was that they're unergonomic which makes them look unwieldy, not confusing?

Reply:
Better? [`70a688d` (#268)](https://github.com/TheBevyFlock/bevy_quickstart/pull/268/commits/70a688d1cbb10aa1a112ca4abb35657e7210bcdf)

Reply:
@benfrankel IME everything concerning exclusive access to `World` is spooky

Reply:
> Better? [`70a688d` (#268)](https://github.com/TheBevyFlock/bevy_quickstart/pull/268/commits/70a688d1cbb10aa1a112ca4abb35657e7210bcdf)

More accurate yes. I still think the doc comment calling the function a `['Command']` is sufficient.

Reply:
@alice-i-cecile, could you act as a tie-breaker here?

Reply:
I think erring on the side of more verbose here is good.